### PR TITLE
Comments crash and misc fixes

### PR DESCRIPTION
**Pull request by ktprograms**
- Fix crash getting channel to load reactions with when signed out.
- Add additional checks for no selected or loaded channels.
- Fix comments not showing due to wrongly filtering blocked channels.

Fix: #369

### CODE REVIEW

File: Odysee/Controllers/Content/CommentsViewController.swift [hunk 1]
```diff
@@ -503,8 +513,8 @@ class CommentsViewController: UIViewController, UITableViewDelegate, UITableView
             return
         }
 
-        if channels.count == 0 {
-            showMessage(message: String.localized("You need to create a channel before you can react to comments"))
+        if currentCommentAsIndex == -1 || channels.count == 0 {
+            showMessage(message: String.localized("You need to select a channel to react to comments"))
```
Comment (akinwale) [inline Odysee/Controllers/Content/CommentsViewController.swift:517]: We need to show a different message if `channels.count == 0` which lets the user know that they should create a channel.

**Review by akinwale — CHANGES_REQUESTED** [review]
Just one minor change required. Everything else looks good, thanks.

File: Odysee/Controllers/Content/CommentsViewController.swift (hunk 1 above)
Comment (ktprograms) [reply Odysee/Controllers/Content/CommentsViewController.swift]: Done

**Review by akinwale — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**